data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -25,7 +27,7 @@
         {% endif -%}
 
         {% if firstSegments.PR1 -%}
-            {% assign pr1Id = firstSegments.PR1 | generate_uuid -%}
+            {% capture pr1Id -%}{% include 'ID/Procedure' with PR1: firstSegments.PR1, baseId: patientId -%}{% endcapture -%}
```

Comment:
Maybe it's more consistent to keep ID related variables in the same naming style, always using the `${resourceType}Id` or `${v2SegmentType}Id`. If we choose the `resourceType`, the ID var name here should be `procedureId`, or `pr1Id` if we choose the later one style.

Reply:
Agree. It is fixed in the update.